data/repositories.xml

```diff
@@ -60,5 +59,8 @@
     </phar>
     <phar alias="sshman" composer="mwender/sshman">
         <repository type="github" url="https://api.github.com/repos/mwender/sshman/releases" />
-    </phar>    
+    </phar>
+    <phar alias="php-coveralls" composer="php-coveralls/php-coveralls">
+        <repository type="github" url="https://api.github.com/repos/php-coveralls/php-coveralls/releases" />
+    </phar>
```

Comment:
I must admit I have no clue what is the pattern of ordering items here... I added new one as last one

Reply:
No worries.